pkg/extensions/enterprise_imports.go

```diff
@@ -29,18 +29,8 @@ import (
 	_ "github.com/spf13/cobra" // used by the standalone apiserver cli
 	_ "github.com/stretchr/testify/require"
 	_ "golang.org/x/time/rate"
-	_ "k8s.io/api"
-	_ "k8s.io/kube-aggregator/pkg/apiserver/scheme"
-	_ "k8s.io/kube-aggregator/pkg/generated/openapi"
-	_ "k8s.io/kube-aggregator/pkg/registry/apiservice/rest"
-	_ "sigs.k8s.io/randfill"
 	_ "xorm.io/builder"
 
-	_ "github.com/grafana/authlib/authn"
-	_ "github.com/grafana/authlib/authz"
-	_ "github.com/grafana/authlib/cache"
-	_ "github.com/grafana/authlib/grpcutils"
-	_ "github.com/grafana/authlib/types"
```

Comment:
The changes here should not be intended, these are all imported in Enterprise codebase.